onnx/backend/test/cpp/driver/gtest_utils.h

```diff
@@ -0,0 +1,4 @@
+#include "gtest/gtest.h"
+#include "test_driver.h"
+
+extern std::vector<ProtoTestCase> all_test_cases;
```

Comment:
Is this some quirky requirement from gtest?

Reply:
It is hard to pass these data from our backend to gtest. There I set this global var for both side to comunicate. Gtest has similar solution of using GFLAG, and it does the same thing of using a gloabl var.